caffe2/core/operator.h

```diff
@@ -613,6 +616,18 @@ class Operator : public OperatorBase {
     return OperatorBase::template Output<Tensor>(idx, type);
   }
 
+  Tensor* OutputTensorCopyFrom(
```

Comment:
This looks dead. Did you mean to put this in a different diff?

Reply:
OK, will put in a different diff